aws/resource_aws_codebuild_project.go

```diff
@@ -157,6 +157,7 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
 								codebuild.EnvironmentTypeLinuxContainer,
+								codebuild.EnvironmentTypeWindowsContainer,
```

Comment:
Can you please add this to the documentation as well? https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/docs/r/codebuild_project.html.markdown#L179